code/game/turfs/open.dm

```diff
@@ -7,17 +7,27 @@
 	var/list/archdrops
 	var/wet
 
+	var/footstep = null
+
 /turf/open/ComponentInitialize()
 	. = ..()
 	if(wet)
 		AddComponent(/datum/component/wet_floor, wet, INFINITY, 0, INFINITY, TRUE)
 	if(LAZYLEN(archdrops))
 		AddComponent(/datum/component/archaeology, archdrops)
 
+/turf/open/Entered(var/mob/living/LM)
```

Comment:
Just to make this point formally, this is not ok. If you want to hook mob movement, use /mob/Moved(). Having the turf check if what entered it is a living mob and all that is nonsense

Reply:
I'll make this into a component